src/bssl.rs

```diff
@@ -47,3 +63,24 @@ macro_rules! bssl_test {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    mod result {
+        use {bssl, c};
+        use std::mem;
```

Comment:
use `core::mem` instead of `std::mem`. In general we use `core` instead of `std` whenever possible.